optashift-employee-rostering-webapp/pom.xml

```diff
@@ -28,135 +30,159 @@
+      <artifactId>resteasy-client</artifactId>
       <version>${arquillian.version.resteasy}</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>org.jboss.resteasy</groupId>
-      <artifactId>resteasy-jaxb-provider</artifactId>
+      <artifactId>resteasy-jaxrs</artifactId>
       <version>${arquillian.version.resteasy}</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>org.jboss.resteasy</groupId>
       <artifactId>resteasy-jackson2-provider</artifactId>
       <version>${arquillian.version.resteasy.jackson2-provider}</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
       <version>${arquillian.version.jackson}</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
       <version>${arquillian.version.jackson}</version>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-jaxb-annotations</artifactId>
+      <version>${arquillian.version.jackson}</version>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.jaxrs</groupId>
       <artifactId>jackson-jaxrs-base</artifactId>
       <version>${arquillian.version.jackson-jaxrs}</version>
       <scope>test</scope>
     </dependency>
-
+    
+   <dependency>
+     <groupId>dom4j</groupId>
+     <artifactId>dom4j</artifactId>
+     <scope>test</scope>
+   </dependency>
+   
+   <dependency>
+      <groupId>org.wildfly.core</groupId>
+      <artifactId>wildfly-launcher</artifactId>
+      <version>${arquillian.version.wildfly}</version>
+      <scope>test</scope>
+    </dependency>
     <dependency>
-      <groupId>com.fasterxml.jackson.module</groupId>
-      <artifactId>jackson-module-jaxb-annotations</artifactId>
-      <version>${arquillian.version.jackson}</version>
+      <groupId>org.wildfly.core</groupId>
+      <artifactId>wildfly-controller-client</artifactId>
+      <version>${arquillian.version.wildfly}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.wildfly.core</groupId>
+      <artifactId>wildfly-protocol</artifactId>
+      <version>${arquillian.version.wildfly}</version>
+      <scope>test</scope>
+    </dependency>
+
   </dependencies>
 
   <profiles>
     <profile>
-      <!-- Use this profile for any OpenShift specific customization your app will need. -->
-      <!-- By default that is to put the resulting archive into the 'deployments' folder. -->
+      <!-- Use this profile for any OpenShift specific customization your 
+        app will need. -->
+      <!-- By default that is to put the resulting archive into the 'deployments' 
+        folder. -->
```

Comment:
Either it's 1 big comment (so one <!-- and one -->), or it's 1 comment per line. Not 2 comments of 2 lines.